### PR TITLE
Log the HTTP header "referer" only if available

### PR DESCRIPTION
**Pull request by zazabe**
_Found this bug when I tried to use the the logger for some debugging..._

### CODE REVIEW

**Comment by zazabe** [comment]
@fvovan please review

File: library/CM/Log/Formatter/Text.php [hunk 1]
```diff
@@ -29,7 +30,7 @@ public function renderContext(CM_Log_Record $record) {
                 'proto'   => isset($server['SERVER_PROTOCOL']) ? $server['SERVER_PROTOCOL'] : '',
                 'host'    => $httpRequest->getHost(),
                 'ip'      => $httpRequest->getIp(),
-                'referer' => $httpRequest->getHeader('referer'),
+                'referer' => isset($headers['referer']) ? $headers['referer'] : '',
```
Comment (fvovan) [inline library/CM/Log/Formatter/Text.php]: It's better to check it with `$httpRequest->hasHeader('referer')` rather using arrays directly.

**Comment by zazabe** [comment]
@fvovan is it better like that?

**Comment by fvovan** [comment]
Yes, it's better but for simplicity i would leave formatting template the same and just  modify  former line 33
`'referer' => $httpRequest->hasHeader('referer')? $httpRequest->getHeader('referer') : '',`

**Comment by zazabe** [comment]
ok, as you want ;)

**Comment by fvovan** [comment]
lgtm!

**Pull request merged**